BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -207,6 +210,9 @@ extension BlueprintView {
                     }
                     layoutTransition.perform {
                         child.layoutAttributes.apply(to: controller.view)
+
+                        contentView.insertSubview(controller.view, at: index)
```

Comment:
This just has the effect of moving the existing subview to the right index, right?

Reply:
That's the goal, yes.